libsplinter/src/biome/key_management/mod.rs

```diff
@@ -21,10 +21,10 @@ use store::diesel::models::KeyModel;
 // Represents a public and private key pair
 #[derive(Serialize)]
```

Comment:
Does this struct still need to be `Serialize`?

Reply:
No, it does not. Addressed.